src/game_updater/GameUpdater.cpp

```diff
@@ -167,28 +168,58 @@ void GameUpdater::UpdatePlayer(std::shared_ptr<GameState> game_state) {
   }
 }
 
+std::shared_ptr<Platform> GetCurrentPlatform(std::shared_ptr<Level> level) {
+  return level->getPlatforms()->at(std::max(
+      0l,
+      std::min((long)(level->getMusic()->getPlayingOffset().asMilliseconds() /
+                      (float)MS_PER_PLATFORM),
+               (long)level->getPlatforms()->size() - 1)));
+}
```

Comment:
The GetCurrentPlatfrom logic is going out the window in my change. That said should the camera depend on the anatomy of the level? I think it might be better  to have the camera follow the player since there's not always going to be a platform to get. The other solution would be to have the level contain data for the camera, for now though I think we should get rid of this and have the camera be at a fixed Y above the player.
If you can do that or if you have another idea let me know @Matthew-Stewart